src/main.c

```diff
@@ -15,6 +16,7 @@ gameState* initGameState() {
 
 int main() {
     gameState* game = initGameState();
+    timer* timer = initTimer();
```

Comment:
Better put 'timer* timer' insight [gameState it self](https://github.com/Maksasj/dungeon-master/blob/master/src/gameState/gameState.h) and 'initTimer();' place in 'initGameState()'
That about the rest? Everything looks great 🤡